working-with-components.md

```diff
@@ -0,0 +1,48 @@
+# Working with components
+
+## What are components
+Components can be thought of as sections of user interface on our websites. There are small components, such as buttons and inputs, and larger components such as product tiles and search headers. Typically larger components will be _composed_ of many smaller components. This pattern of designing many small components to compose together for larger reuseable components within a page is known as **atomic design**. To read more on Atomic Design see [Brad Frost's excellent post](http://bradfrost.com/blog/post/atomic-web-design/), but for the purposes of working with components at Hx we'll be referring to three of the 5 atomic design component levels:
```

Comment:
I think we should also use an example of none text based component. Like Map etc.

Maybe not needed, but I would like to get across the message, everything can be a component.